src/model/adapter.py

```diff
@@ -60,14 +60,12 @@ def add_prefix(x, prefix):
     if len(object_ids) == 0:
         return {}
     query = json.dumps({'ids': object_ids, 'dbFrom': db_from, 'dbTo': db_to, 'type': 'Metabolite'})
-    start_time = time.time()
-    LOGGER.info('query id mapper at %s with %s', ID_MAPPER_API, str(query))
-    async with aiohttp.ClientSession() as session:
-        async with session.post(ID_MAPPER_API, data=query) as r:
-            assert r.status == 200, f'response status {r.status} from identifier service'
-            result = await r.json()
-            LOGGER.info('id mapper call took %s', time.time() - start_time)
-            return result['ids']
+    logger.info('query id mapper at %s with %s', ID_MAPPER_API, str(query))
+    with log_time(operation=f"ID map request for ids: {object_ids}"):
+        async with aiohttp.ClientSession() as session:
```

Comment:
I'm not sure if it's possible with async `with`-s, but did you look into listing the with block instead of nesting them? https://stackoverflow.com/a/27551161/2419215

Reply:
That's a nice visual improvement, thanks